speedcenterRun

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+BIN_DIR="$(dirname "$0")/build/install/gentle/bin"
```

Comment:
```suggestion
BIN_DIR="$(dirname "$0")/build/install/gentle/bin"
cd "$(dirname "$0")"
```
We need to either `cd` there or our java side needs to handle this. File paths are relative *to the path of the executable* and not the CWD.

Reply:
Nope.